android/daemon/tunnel/libwg-go/Makefile

```diff
@@ -45,6 +45,7 @@ $(BUILDDIR)/go-$(GO_VERSION)/.prepared: $(GRADLE_USER_HOME)/caches/golang/$(GO_T
 	patch -p1 -f -N -r- -d "$(dir $@)" < goruntime-boottime-over-monotonic.diff && \
 	touch "$@"'
 
+$(DESTDIR)/libwg-go.so: export GOROOT := $(BUILDDIR)/go-$(GO_VERSION)/
```

Comment:
This makes sure if you have another go env active... like conda, we still pickup the patched version this makefile creates :)